me.amankhanna.opendeck.yaml

```diff
@@ -0,0 +1,35 @@
+id: me.amankhanna.opendeck
+
+runtime: org.gnome.Platform
+runtime-version: "46"
```

Comment:
47 is the current version

Reply:
Started this a few days ago when it wasn't, apologies.

Reply:
a few days ago it was already the current version. Has been for a few weeks now (September)

Reply:
Maybe I just took "not found" to mean not existing, not not installed. It's fixed now anyway